video/bink_decoder.cpp

```diff
@@ -1086,15 +1083,22 @@ void BinkDecoder::BinkVideoTrack::readDCTCoeffs(VideoFrame &video, int32 *block,
 	int coefIdx[64];
 
 	int listStart = 64;
-	int listEnd   = 64;
-
-	int coefList[128];      int modeList[128];
-	coefList[listEnd] = 4;  modeList[listEnd++] = 0;
-	coefList[listEnd] = 24; modeList[listEnd++] = 0;
-	coefList[listEnd] = 44; modeList[listEnd++] = 0;
-	coefList[listEnd] = 1;  modeList[listEnd++] = 3;
-	coefList[listEnd] = 2;  modeList[listEnd++] = 3;
-	coefList[listEnd] = 3;  modeList[listEnd++] = 3;
+	int listEnd = 64;
+
+	int coefList[128];
```

Comment:
This also ruined the authors' idea.

I'm stopping here. Could you please extend the rules, so we have fewer changes, such as #include sorting, allowing several statements on the same line, allowing optional alignment etc.